src/editor.c

```diff
@@ -4968,6 +4990,7 @@ static ScintillaObject *create_new_sci(GeanyEditor *editor)
 		g_signal_connect(sci, "button-press-event", G_CALLBACK(on_editor_button_press_event), editor);
 		g_signal_connect(sci, "scroll-event", G_CALLBACK(on_editor_scroll_event), editor);
 		g_signal_connect(sci, "motion-notify-event", G_CALLBACK(on_motion_event), NULL);
+		g_signal_connect(sci, "motion-notify-event", G_CALLBACK(motion_notify_event), editor);
```

Comment:
shouldn't the two `motion-notify-event` handlers be merged?

Reply:
`on_motion_event()` is in some other file than `editor.c` and probably needs access to some stuff defined there but I could call it from within `motion_notify_event()` so the signal is connected just once.

Reply:
Ah right, the other one is the generic "let's focus stuff" thingy.  But yeah, probably having a single handler would be better -- although it doesn't change a thing save confusing me a tad :slightly_smiling_face: